### PR TITLE
#644 Using LoggingInput without providing a Logger does not work

### PR DESCRIPTION
**Pull request by nikialeksey**
Retrieve logger level from parent if it inherits

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#715](https://github.com/yegor256/cactoos/issues/715) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#715](https://github.com/yegor256/cactoos/issues/715) is assigned to @neonailol[/z](https://www.0crat.com/u/neonailol), here is [why](https://www.0crat.com/footprint/C63314D6Z/591743650). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/java/org/cactoos/io/LoggingInputStream.java [hunk 1]
```diff
@@ -213,12 +213,29 @@ public boolean markSupported() {
...
         );
         return supported;
     }
+
+    /**
+     * Retrieve {@code logger} level.
+     *
+     * @return Level of {@code logger}.
+     */
+    private Level level() {
```
Comment (neonailol) [inline src/main/java/org/cactoos/io/LoggingInputStream.java]: I think this part should be inside sticky scalar

**Comment by 0crat** [comment]
@neonailol[/z](https://www.0crat.com/u/neonailol) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

File: src/main/java/org/cactoos/io/LoggingInputStream.java (hunk 1 above)
Comment (nikialeksey) [reply src/main/java/org/cactoos/io/LoggingInputStream.java]: Done in [`954e7ee`](https://github.com/yegor256/cactoos/pull/715/commits/954e7eeebe7df20005d4d075254d38379fdd3251)

**Comment by neonailol** [comment]
@nikialeksey thanks, all good now

**Comment by neonailol** [comment]
@rultor good to merge

**Comment by 0crat** [comment]
@neonailol[/z](https://www.0crat.com/u/neonailol) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by neonailol** [comment]
@yegor256 Please confirm merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @neonailol[/z](https://www.0crat.com/u/neonailol), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#715](https://github.com/yegor256/cactoos/issues/715) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by elenavolokhova** [comment]
@neonailol According to our [QA rules](http://datum.zerocracy.com/pages/policy.html#42)
> The code reviewer found at least three problems in the code.

Only one issue was found during review.
Please, **confirm** that you'll try to find at least three design problems in next reviews.

**Comment by elenavolokhova** [comment]
@neonailol I need your confirmation in order to complete my review.

**Comment by neonailol** [comment]
@elenavolokhova i confirm

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 points just awarded to @neonailol[/z](https://www.0crat.com/u/neonailol)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Pull request merged**